core/src/main/java/jenkins/views/Header.java

```diff
@@ -0,0 +1,13 @@
+package jenkins.views;
+
+import hudson.ExtensionPoint;
+
+public interface Header extends ExtensionPoint {
```

Comment:
Needs Javadoc for the new extension point

Reply:
Included in https://github.com/jenkinsci/jenkins/pull/5909/commits/88eac160e4ec72ee1a60680d1a8f36da8c79242e